### DISCUSSION TITLE
TMS optimization

### DISCUSSION THREAD

**Discussion by nuan750** [simnibs/simnibs#277] [8. Simulation workflows]
May I ask how long does it take to run a TMS optimizaion using ADM method? I am trying it on the sample data, but it still not completed after 1 hour, is it normal? Thanks

**Comment by TorgeW** [comment]
Depending on your system and the specific optimization problem the time for the optimization can vary a lot.
Has it finished yet?

If you are in doubt you can supply more information so that we can give you a suitable answer.
